stack2stack.py

```diff
@@ -93,27 +94,55 @@ def migrate_images():
     new_cloud_glance_client = glance_client.Client('1', token=new_cloud_keystone_client.auth_token, endpoint=new_endpoint.publicurl)
     images = old_cloud_glance_client.images.list()
     for i in images:
-        print 'Found image %s' % i.name
+        print ''
+        print 'Old Cloud: Found image           - %s' % i.name
+        print '           Image ID              - %s' % i.id
+        try:
+            print '           Image       Type      - %s' % i.properties['image_type']
+        except KeyError:
+            print '           Image       Type      - image'
```

Comment:
This threw an attribute error for 'has_key'

Old Cloud: Found image           - Windows 8.1 Ent x64
           Image ID              - a9f1446f-f93b-4710-830e-56f67254197e
Traceback (most recent call last):
  File "./qeos2centralci.py", line 194, in <module>
    main()
  File "./qeos2centralci.py", line 190, in main
    migrate_images()
  File "./qeos2centralci.py", line 107, in migrate_images
    if i.has_key('image_type'):
  File "/usr/lib/python2.7/site-packages/glanceclient/openstack/common/apiclient/base.py", line 494, in __getattr__
    raise AttributeError(k)
AttributeError: has_key